modules/linear_boltzmann_solvers/a_lbs_solver/source_functions/adjoint_src_function.cc

```diff
@@ -14,10 +14,10 @@ AdjointSourceFunction::AdjointSourceFunction(const LBSSolver& lbs_solver)
 }
 
 void
-AdjointSourceFunction::AddVolumetricQOISources(LBSGroupset& groupset,
-                                               std::vector<double>& destination_q,
+AdjointSourceFunction::AddVolumetricQOISources(const LBSGroupset& groupset,
+                                               std::vector<double>& q,
```

Comment:
what is q here? A quadrature point? a source? A QoI value? I am a bit lost.

Reply:
The source vector being contributed to by the source function. We could call this something different, maybe `b` or `dest` if you'd like.